docs/install.md

```diff
@@ -348,6 +348,45 @@ Select the following tabs for the installation description of your target.
 
     ![Parallels Desktop DietPi startup](assets/images/Parallels2.png){: width="640" height="273" loading="lazy"}
 
+=== "UTM (macOS)"
+
+    <font size="+2">Introduction</font>
+
+    Virtual machine images are great for those occasions where you want to set up a DietPi system very quickly and test things. Also it may be used as a Debian based Linux system with a small footprint for development purposes, e.g. with the X11 window system. The small footprint makes it optimally usable on PCs without a huge built in RAM. Also several VMs may be run for different applications.
+
+    One big advantage of such a VM is that it needs only a couple of minutes coming to a running DietPi system.
+
+    One of the options of a virtual machine is [__UTM__](https://mac.getutm.app/) for macOS.
+
+    ![UTM Desktop DietPi machine](assets/images/UTM1.png){: width="900" height="506" loading="lazy"}
+
+    <font size="+2">Prerequisites</font>
+
+    As a starting point you need an **Apple Mac with a running UTM software** on which the DietPi system will run (x86 system, e.g. Mac mini 2011/2012/2014/2018 as well as M1 Apple silicon).  
+    On this MAC a free harddisk space of about  
+
+    - 1.2 GiB for a minimal running system  
+    - 5 - 10 GiB for a typical running system with X11  
+
+    is needed. A recommended size is at least a free space of 10 GiB.
+
+    <font size="+2">1. Download and extract the DietPi disk image</font>
+
+    Download the **DietPi UTM** image from [dietpi.com](https://dietpi.com/#download){:class="nospellcheck"}.
+
+    Double click on the downloaded file to extract it (or via option "Open in Finder"). Copy/Move the contained `.utm` file to the UTM virtual machine directory. This is typically located within the (hidden) user subdirectory `.../Library/Containers/com.utmapp.UTM/Data/Documents`.
+
+    <font size="+2">2. Import the virtual machine in UTM</font>
+
+    As next, start resp. restart the UTM program. The imported DietPi virtual machine should be visible there. Then select the imported machine to start it up.
+
+    ![UTM DietPi startup](assets/images/UTM2.png){: width="640" height="517" loading="lazy"}
+
+    <font size="+2">3. UTM machine for M1 (Apple silicon)</font>
+
+    The provided UTM VM is generated for x86 systems. In case of a needed VM for Apple's M1 architecture, a new machine needs to be generated. This can be achieved by adding a new UTM VM via the `+` option in the UTM program. Then a new VM can be generated by using the DietPi images for Native PCs. Refer to the UTM documentation how to do this.  
```

Comment:
The native PC images are `x86_64`, so they won't work. Users need to start with an ARM Debian system, at best using the Debian `mini.iso` installer.

Reply:
Not really. UTM distinguishes between *virtualization* and *emulation*.
*Emulation* gives the opportunity to create a VM on a M1 Mac based on a x86 base image. 
E.g. there: 

- https://mac.getutm.app/: _UTM employs Apple's Hypervisor virtualization framework to run ARM64 operating systems on Apple Silicon at near native speeds. On Intel Macs, x86/x64 operating system can be virtualized. In addition, lower performance emulation is available to run x86/x64 on Apple Silicon as well as ARM64 on Intel._
- https://newsbeezer.com/germanyeng/virtualizer-and-emulator-utm-comes-to-the-mac-app-store/
- https://levelup.gitconnected.com/running-amd64-linux-on-apple-m1-with-qemu-utm-64d67cccd6f8
- https://www.heise.de/news/Virtualisierer-und-Emulator-UTM-kommt-in-den-Mac-App-Store-5070019.html, see "Virtualisierung und Emulation – auch für M1-Macs".

During the wizard to create a new VM there is the question for virt vs. emu.

- https://www.youtube.com/watch?v=9pe2u3OHfwo: Goto 3:25
- https://github.com/utmapp/UTM/wiki/Install-Windows-ARM64-on-Apple-M1

Reply:
Ah okay, that makes sense, since it uses QEMU under the hood which supports both. It should be however made clear than that running an `x86_64` image on an M1 ARM system via emulation will provide pretty bad performance. So it works, great for testing and light server tasks, but anything CPU intense is not advised to run like that. So it is then indeed better to start with a fresh Debian `aarch64`/`arm64` image or installer and run PREP. Actually native M1 support has been added to Linux with v5.13, so I wonder whether the v5.10 kernel works, i.e. whether the M1 is "backwards" compatible 🤔.